docs/es/week12/12-3.md

```diff
@@ -0,0 +1,546 @@
+Sin embargo, aún es posible transformar los valores multi-cabeza para que tengan la dimensión original $\R^d$ usando un $\vect{W_h} \in \mathbb{R}^{d \times hd}$. Esta es sólo una de las posibles maneras de implementar el almacén de clave/valor.
+
+
+<!-- ## [The Transformer](https://www.youtube.com/watch?v=f01J0Dri-6k&t=2114s) -->
+## [El Transformador](https://www.youtube.com/watch?v=f01J0Dri-6k&t=2114s)
+
+<!-- Expanding on our knowledge of attention in particular, we now interpret the fundamental building blocks of the transformer. In particular, we will take a forward pass through a basic transformer, and see how attention is used in the standard encoder-decoder paradigm and compares to the sequential architectures of RNNs. -->
+Con el fin de expandir nuestro conocimiento sobre atención, interpretaremos ahora los bloques fundamentales del transformador. En particular, haremos un recorrido de principio a fin de un transformador básico, y veremos cómo se usa la atención en el paradigma estándar del codificador-decodificador, y cómo se compara esto con las arquitecturas secuenciales de las RNRs (Redes Neuronales Recurrentes).
+
+
+<!-- ### Encoder-Decoder Architecture -->
+### Arquitectura Codificador-Decodificador
+
+<!-- We should be familiar with this terminology. It is shown most prominently during autoencoder demonstrations, and is prerequisite understanding up to this point. To summarize, an input is fed through an encoder and decoder which impose some sort of bottleneck on the data, forcing only the most important information through. This information is stored in the output of the encoder block, and can be used for a variety of unrelated tasks. -->
+Todos deberíamos estar familiarizados con esta terminología que se presenta de manera más prominente durante demostraciones del auto-codificador, y es un prerequisito conceptual en esta parte del curso. En resumen, el codificador-decodificador recibe una entrada e impone una clase de cuello de botella sobre la misma, forzando a que sólo la información más importante pase a través del mismo. Esta información se almacena en la salida del codificador, y puede ser utilizada para una variedad de tareas no relacionadas.
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-3/figure1.png" style="zoom: 60%; background-color:#DCDCDC;" /><br>
+<!-- <b>Figure 1:</b> Two example diagrams of an autoencoder. The model on the left shows how an autoencoder can be design with two affine transformations + activations, where the image on the right replaces this single "layer" with an arbitrary module of operations. -->
+<b>Figura 1:</b> Dos diagramas ejemplificando un auto-codificador. El modelo de la izquierda muestra cómo un auto-codificador se puede diseñar con dos transformaciones afines + activaciones, mientras que el modelo de la derecha reemplaza esta "capa" única con un módulo arbitrario de operaciones.
```

Comment:
```suggestion
<b>Figura 1:</b> Dos diagramas ejemplificando un autoencoder. El modelo de la izquierda muestra cómo un auto-codificador se puede diseñar con dos transformaciones afines + activaciones, mientras que el modelo de la derecha reemplaza esta "capa" única con un módulo arbitrario de operaciones.
```
I saw in the translation for [lesson 7-3](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/es/week07/07-3.md) that _autoencoder_ was not translated to Spanish.  
We should either review lesson 7 or use the English name here to keep the notation consistent. What do you think?

Reply:
In this case, I feel like "auto-codificador" is a good choice (unless we agreed to never translate names of technical terms like that). Maybe a good discussion to have in Slack.

Reply:
Sounds good to me!